src/DeferredResult/DeferredResult.ts

```diff
@@ -154,3 +154,51 @@ export const matchOrElse =
                 return assertExhaustive(deferredResult)
         }
     }
+
+/**
+ * Maps the wrapped `Resolved` value when `Result` is `Ok` using the given function.
+ * Passes through `Resolved` when `Result` is `Err`, `InProgress`, and `NotStarted` as-is.
+ *
+ * @group Mapping
+ *
+ * @example
+ * pipe(
+ *     Deferred.resolved(Result.ok("cheese")),
+ *     DeferredResult.map(s => s.length),
+ *     DeferredResult.matchOrElse({
+ *         resolvedOk: r => r,
+ *         orElse: 0
+ * 	    })
```

Comment:
⛏️ on the tab/spacing here as well in this example

Reply:
https://github.com/fp-toolkit/fp-toolkit/pull/66/commits/56461bc3042a4c0f1f28ced77ad111edc3816758